src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/dataBrokerProfiles.module.scss

```diff
@@ -0,0 +1,188 @@
+@import "../../../../../../../tokens";
+
+.contentWrapper {
+  max-width: $content-md;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.buttonsWrapper {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+
+  a {
+    display: block;
+  }
+
+  gap: $spacing-md;
+
+  @media screen and (min-width: $screen-sm) {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+  }
+}
+
+.upgradeToggleWrapper {
+  display: block;
+  margin-bottom: $spacing-lg;
+
+  > span {
+    display: block;
+  }
+}
+
+.removalStepsList {
+  font: $text-body-md;
+  text-align: left;
+
+  li {
+    margin-bottom: $spacing-md;
+
+    > strong {
+      display: block;
+      margin-bottom: $spacing-xs;
+      font-weight: 700;
+    }
+
+    // The subhead/bullet content is wrapped in a span to style to OL marker (1.)
+    > span {
+      font: $text-body-sm;
+      font-weight: normal;
+    }
+  }
+}
+
+.content {
+  text-align: center;
+  max-width: $content-md;
+  margin: 0 auto $spacing-2xl;
+
+  p,
+  h3 {
+    margin-block-end: $spacing-md;
+  }
+
+  h3 + p {
+    text-align: left;
+  }
+
+  > a {
+    color: $color-blue-60;
+    display: inline-block;
+    margin-inline-start: $spacing-sm;
+  }
+}
+
+.cityScape {
+  margin: 0 auto $spacing-2xl;
+  display: block;
+  max-width: 100%;
+}
+
+// TODO: Replace with @include question-mark-circle-btn;
+.questionTooltipWrapper {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: $spacing-xs;
+  font: $text-title-3xs;
+}
+
+.questionTooltip {
+  appearance: none;
+  background-color: transparent;
+  border: 0;
+  cursor: pointer;
+  // TODO: Add hover/focus state
+}
+
+.contentAutomaticRemove {
+  h3 {
+    text-align: left;
+  }
+}
+
+.featuresList {
+  text-align: left;
+  max-width: $content-sm;
+  margin-bottom: $spacing-lg;
+
+  ul {
+    padding: 0;
+    li {
+      background: url("../images/icon-check-mark.svg") no-repeat left center;
+      padding: $spacing-sm 0 $spacing-sm $spacing-xl;
+      list-style: none;
+      margin: 0;
+      vertical-align: middle;
+    }
+  }
+}
+
+.upgradeToggle {
+  width: auto;
+  display: inline-flex;
+  gap: $spacing-sm;
+  background-color: $color-grey-10;
+  border-radius: $border-radius-xl;
+  padding: $spacing-sm $spacing-sm;
+  font: $text-body-sm;
+  margin-bottom: $spacing-md;
+
+  button {
+    cursor: pointer;
+    appearance: none;
+    background-color: transparent;
+    border: 0;
+    color: $color-grey-40;
+    padding: $spacing-xs $spacing-md;
+
+    &.isActive,
+    &:hover {
+      background-color: $color-white;
+      border-radius: $border-radius-xl;
+      color: $color-informational;
+    }
+  }
+}
+
+.upgradeContentWrapper {
+  @media screen and (min-width: $screen-md) {
+    display: flex;
+    gap: $spacing-md;
+    justify-content: space-between;
+    // Used to stop purple box from strecthing full width
```

Comment:
```suggestion
    // Used to stop purple box from stretching full width
```

Reply:
😅  Revised in 67ecbc1513e9465bc8032d4293a452706739913b